### PR TITLE
feat: create message interface migration

### PR DESCRIPTION
**Pull request by luizmuraro**
Description: simple migration strategy to avoid breaking old user apps.

### CODE REVIEW

File: www/src/states/use-chat-state.ts [hunk 1]
```diff
@@ -62,6 +62,43 @@ const createAssistantMessage = (chatId: string): Message => {
   );
 };
 
+// Migrate function, should be removed in the future
```
Comment (ribeirojose) [inline www/src/states/use-chat-state.ts:65]: can you add a task on Linear for us to remove this and add a TODO comment with its reference here?

File: www/src/states/use-chat-state.ts [hunk 2]
```diff
@@ -187,6 +224,13 @@ const useChatStoreBase = create<ChatStore>()(
     {
       name: "chat-storage",
       getStorage: () => localStorage,
+      migrate: (persistedState: any, version: number) => {
```
Comment (ribeirojose) [inline www/src/states/use-chat-state.ts:227]: 😮 zustand ❤️  cc @devjoaov you'll enjoy this!

have you been able to test this @luizmuraro ? I can help with that

**Review by ribeirojose — APPROVED** [review]
👌

File: www/src/states/use-chat-state.ts (hunk 2 above)
Comment (luizmuraro) [reply www/src/states/use-chat-state.ts:227]: I've tested it going to the main branch, creating some messages there and then, after coming back here, everything was migrated! But would be great to make some other tests!

**Pull request merged**